### PR TITLE
add Chromium item to Edge troubleshooting

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[433f174](https://github.com/OfficeDev/office-js-docs-pr/commits/433f1746610cf62eedceb78c8efc08fe323ac171)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/chromium/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002072257598611-1586%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=dgubVzFeb3RIGOt%2F0i2vzOqILeYNQCERfPHJ9oX6OvQ%3D&st=2020-02-07T22%3A54%3A11Z&se=2020-03-09T22%3A59%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/concepts/browsers-used-by-office-web-add-ins.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ The following table shows which browser is used for the various platforms and op
 
 ## Troubleshooting Microsoft Edge Issues
 
+### Chromium-based Edge is installed on my development computer, but my add-in does not use it
+
+The base browser in [Microsoft Edge](https://support.microsoft.com/en-us/help/4501095/download-the-new-microsoft-edge-based-on-chromium) has changed to Chromium. But the older base, called EdgeHTML, is not removed when Chromium-based Edge is installed. Office will still use the EdgeHTML base for add-ins until a build of Office 365 that supports Chromium is installed on the computer. We expect these builds to ship in 2020. They will likely appear in the Insiders channel in the first half of the year.
```
Comment (AlexJerabek) [inline docs/concepts/browsers-used-by-office-web-add-ins.md]: Please remove the localization string ("en-us") from the URL.

Comment (AlexJerabek) [inline docs/concepts/browsers-used-by-office-web-add-ins.md]: The second sentence "But the older base, called EdgeHTML, is not removed when Chromium-based Edge is installed." seems unnecessary for an add-in developer to know. Can we just remove it?

If so, change the next sentence a bit to indicate EdgeHTML is the old version ("Office will still use Edge's old base, EdgeHTML, for add-ins...").

**Comment by o365devx** [comment]
Docs Build status updates of commit _[37b294b](https://github.com/OfficeDev/office-js-docs-pr/commits/37b294b091f1fc17d90c5c18f0c050b5658227aa)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/chromium/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002080109357515-1586%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=zYfyPko3wDZjTOOBfYz7Vri2hS8vwovw5DMbzR5EfMw%3D&st=2020-02-08T01%3A05%3A46Z&se=2020-03-10T01%3A10%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/concepts/browsers-used-by-office-web-add-ins.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/concepts/browsers-used-by-office-web-add-ins.md]: @Rick-Kirkham  Fair enough. Can we remove the "But" from the beginning of that sentence ("The older base...")?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[124879f](https://github.com/OfficeDev/office-js-docs-pr/commits/124879f89508bb835bb29ef621c8693c95382434)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/chromium/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002101757193807-1586%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=EZcTDE1w62NEiCfdQJUwslhhAD8o8U1F0uc7pwOYoXs%3D&st=2020-02-10T17%3A53%3A32Z&se=2020-03-12T17%3A58%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**